open_seq2seq/data/speech2text/speech_utils.py

```diff
@@ -337,10 +407,28 @@ def get_speech_features_librosa(signal, sample_freq, num_features,
     num_features].
     audio_duration (float): duration of the signal in seconds
   """
-  if augmentation:
-    signal = augment_audio_signal(signal.astype(np.float32), sample_freq, augmentation)
-  else:
-    signal = normalize_signal(signal.astype(np.float32))
+  if augmentation is not None:
+    prob = np.random.rand()
+    if prob < prob_noise[0]:
+      signal = augment_audio_signal(signal, sample_freq, augmentation)
+    prob = np.random.rand()
+    if prob < prob_noise[1]:
+      signal = aug_with_pitch(signal, sample_freq)
+    if custom_noise is not None:
+      if 'noise_level_max' not in augmentation:
+        raise ValueError('noise_level_max has to be included in augmentation '
+                         'when augmentation it is not None')
+      if 'noise_level_min' not in augmentation:
+        raise ValueError('noise_level_min has to be included in augmentation '
+                         'when augmentation it is not None')
+      if 'time_stretch_ratio' not in augmentation:
```

Comment:
There is no need to check for `time_stretch_ratio` key in `augmentation`. It is optional.